index.css

```diff
@@ -0,0 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+@import 'https://cocky-easley-f83c76.netlify.app/index.css';
+
+body {
+  overflow-x: hidden;
+  /* font-family: 'Noto Sans', sans-serif;
+font-family: 'Montserrat', sans-serif; */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Montserrat', sans-serif;
```

Comment:
You can add font family to body itself, rather than individual elements.

Reply:
i wanted the fonts to apply to heading elements only 😊